### PR TITLE
Add Autarky link to "Who's using Ink" section

### PR DESCRIPTION
**Pull request by pranshuchittora**
https://github.com/pranshuchittora/autarky

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ Feel free to play around with the code and fork this repl at [https://repl.it/@v
 - [garson](https://github.com/goliney/garson) - Build interactive config-based command-line interfaces.
 - [git-contrib-calendar](https://github.com/giannisp/git-contrib-calendar) - Display a contributions calendar for any git repository.
 - [gitgud](https://github.com/GitGud-org/GitGud) - An interactive command-line GUI for Git.
+- [Autarky](https://github.com/pranshuchittora/autarky) - Liberating disk space from file_folder node_modules.
```
Comment (sindresorhus) [inline readme.md:92]:
```suggestion
- [Autarky](https://github.com/pranshuchittora/autarky) - Liberating disk space from node_modules.
```

And I don't think the description explains clearly enough what it actually does. No marketing talk or tag lines.

Comment (pranshuchittora) [reply readme.md]: Thanks for the review @sindresorhus :+1: 
I have made the respective changes looking forward to your review.

**Comment by vadimdemedes** [comment]
Cool project and nice logo!

**Comment by pranshuchittora** [comment]
> Cool project and nice logo!

Thanks @vadimdemedes it means a lot. And thank you for creating ink 🤩🙏

**Pull request merged**